src/libs/ppscale/ppscale.c

```diff
@@ -0,0 +1,147 @@
+/* Copyright 2018-2020          Anton Shepelev          (anton.txt@gmail.com) */
+/* Usage of the works is permitted  provided that this instrument is retained */
+/* with the works, so that any entity that uses the works is notified of this */
+/* instrument.    DISCLAIMER: THE WORKS ARE WITHOUT WARRANTY.                 */
+
+/* ----------------------- Pixel-perfect scaling unit ----------------------- */
+/* This unit uses the Horstmann indentation style.                            */
+
+#include <math.h>
+#include <string.h>
+
+static double min( double a, double b )
+{	double res;
+	if( a < b ) res = a;
+	else        res = b;
+	return res;	
+}
```

Comment:
How about `double fmin(double, double);` defined in C `math.h`?

Reply:
Indeed, my library is targeting perdantic C89.

Reply:
Understood, resolved :)
(I will respond to other comments and continue review tomorrow).